### PR TITLE
updating documentation to use FSDP2

### PR DESCRIPTION
**Pull request by nagpalar**
### What does this PR do?

 Llama 3+ FSDP Training Example for EKS/Slurm Workshop update 

🛑 Please open an issue first to discuss any significant work and flesh out details/direction. When we triage the issues, we will add labels to the issue like "Enhancement", "Bug" which should indicate to you that this issue can be worked on and we are looking forward to your PR. We would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/ai-on-sagemaker-hyperpod/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

updating the documentation to include steps for FSDP2
### More

- [X] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Mandatory for new blueprints. 
- [ ] Yes, I have added a example to support my blueprint PR.
- [X] Yes, I have updated the `website/docs` section for this feature.

### For Moderators

- [ ] E2E Test successfully complete before merge?
- [ ] I ran the proper security checks explained on the ML Frameworks wiki

### Additional Notes

Requesting  mvincig to check it

### CODE REVIEW

**Review by nagpalar — COMMENTED** [review]
Documentation update on FSDP2

File: website/docs/00-eks-orchestration/training-and-fine-tuning/fsdp/fully-sharded-data-parallel.md [hunk 1]
```diff
@@ -92,7 +102,7 @@ cd awsome-distributed-training/3.test_cases/pytorch/FSDP
 
 ### 1.2 Build a Docker Image
 
-Now we'll build a container image that includes PyTorch, FSDP training code, and all necessary dependencies. First, we need to authenticate with the public ECR registry to access base images.
+Now we'll build a container image that includes PyTorch 2.9.1+ (required for FSDP2), training code, and all necessary dependencies. First, we need to authenticate with the public ECR registry to access base images.
```
Comment (mvinci12) [inline website/docs/00-eks-orchestration/training-and-fine-tuning/fsdp/fully-sharded-data-parallel.md]: if we're using 2.9.1 can we updated docker image tags? right now they seem to be set to using 2.6.0

File: website/docs/00-eks-orchestration/training-and-fine-tuning/fsdp/fully-sharded-data-parallel.md [hunk 2]
```diff
@@ -200,18 +226,32 @@ cd kubernetes/
...
 
-For ml.g5.8xlarge x 8:
+For ml.g5.8xlarge x 2 with elastic training (1-2 nodes):
 
 ```bash
-export IMAGE_URI=${REGISTRY}fsdp:pytorch2.5.1
+export IMAGE_URI=${REGISTRY}fsdp:pytorch2.6.0
 export INSTANCE_TYPE=ml.g5.8xlarge
-export NUM_NODES=8
+export MIN_NODES=1  # Minimum nodes for elastic training
```
Comment (mvinci12) [inline website/docs/00-eks-orchestration/training-and-fine-tuning/fsdp/fully-sharded-data-parallel.md:234]: have we configured this for hpto elastic training?

**Review by mvinci12 — CHANGES_REQUESTED** [review]
